src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,27 @@
+import { User } from '../../types/User';
+
+type Props = {
+  user: User,
+  completed: boolean,
```

Comment:
nit: Consider renaming `completed` prop to `isCompleted` for better readability and to indicate that it's a boolean value. 📘

Reply:
Fix Vadim's comment